### DISCUSSION TITLE
Only the first page of my vault published the other 50+ pages and all other features are missing

### DISCUSSION THREAD

**Discussion by ciconsulting** [flowershow/flowershow#623] [General and Help]
I followed the instructions here: https://flowershow.app/docs/publish-howto 

The little flower icon in the ribbon shows all notes published. The front page has some links to other notes, they're all 404. 

What should I do to debug this? 

Does the link format have to be set to: Shortest path when possible, relative path to file, absolute path in vault or it doesn't matter? 

I noticed that updating a note or adding a new one does not trigger vercel to update the site. The plugin shows the notes as updated/published.

**Comment by rufuspollock** [comment]
@ciconsulting hi there - thanks for reporting an issue. Can you give any screenshots or indication of what pages are not showing up?

**Comment by ciconsulting** [comment]
Only the page (or pages) in the root show up, all other pages don't show up.

The front page has some links to "recent posts" but they are all 404. None of notes actually show up whether linked from the home page or not.

**Reply by rufuspollock** [reply]
@ciconsulting you have the files in the `content` folder? So when you say only root folder show up you mean that folder?
